src/drafts/hooks/useDynamicTextareaHeight.ts

```diff
@@ -56,7 +56,16 @@ export const useDynamicTextareaHeight = ({
     if (minHeightLines !== undefined) setMinHeight(`calc(${minHeightLines} * ${lineHeight})`)
     if (maxHeightLines !== undefined) setMaxHeight(`calc(${maxHeightLines} * ${lineHeight})`)
     // `value` is an unnecessary dependency but it enables us to recalculate as the user types
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [minHeightLines, maxHeightLines, value, elementRef])
 
+  useLayoutEffect(refreshHeight, [refreshHeight])
+
+  // With Slots, initial render of the component is delayed and so the initial layout effect can occur
+  // before the target element has actually been calculated in the DOM. But if we only use regular effects,
+  // there will be a visible flash on initial render when not using slots
```

Comment:
Should we update this comment to not mention slots since the bug wasn't caused by slots?